app.rb

```diff
@@ -2,4 +2,15 @@
 
 class App < Sinatra::Base
 
-end
\ No newline at end of file
+  get '/' do
+    erb :index
+  end
+
+  get '/new' do
+    erb :create_puppy
+  end
+
+  post '/new' do
+    erb :display_puppy
+  end
```

Comment:
It would be better to use the `Puppy` class here. You could initialize it with the params, then have access it in the view.